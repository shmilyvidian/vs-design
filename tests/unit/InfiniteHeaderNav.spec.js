import { shallowMount, mount } from '@vue/test-utils'
import InfiniteHeaderNav from '@/packages/infinite-header-nav/src/index.vue'

const wrapper = mount(InfiniteHeaderNav)

describe('InfiniteEmpty.vue', () => {
  it('shonld render empty-container class', () => {
    expect(wrapper.attributes().class).toMatch('header-wrapper')
    wrapper.destroy()
  })

  it('renders props.type to match class', () => {
    const typeArr = ['error', 'network', 'search']
 
  })

})

