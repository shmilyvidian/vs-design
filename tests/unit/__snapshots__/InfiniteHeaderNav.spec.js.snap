// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Infinite header nav.vue render InfiniteHeaderNav 1`] = `
<header class="vs-header-wrapper" style="color: rgb(255, 255, 255);">
  <div class="vs-header-left"><i class="van-icon van-icon-arrow-left" style="color: orange; font-size: 22px;">
      <!---->
    </i> </div>
  <div class="vs-header-title vs-right-padding">
    <div class="title-content"></div>
  </div>
  <div class="vs-header-right"></div>
</header>
`;
